fio/src/fiodriver/fioman.c

```diff
@@ -726,13 +726,9 @@ fioman_add_def_fiod_frames
 			if (rx_frame)
 				list_add_tail( &((FIOMSG_RX_FRAME *)(rx_frame))->elem, rx_frames );
 
-#if 0
-			/* Indicate other valid frames not sent by default (FIO332,FIOTS1,FIOINSIU,FIOOUTSIU) */
-			p_fiod->frame_frequency_table[60] = FIO_HZ_0;
-			p_fiod->frame_frequency_table[61] = FIO_HZ_0;
+			/* Indicate other valid frames not sent by default (FIOCMU) */
 			p_fiod->frame_frequency_table[62] = FIO_HZ_0;
-			p_fiod->frame_frequency_table[65] = FIO_HZ_0;
-#endif
```

Comment:
Shouldn't we allow 60, 61, 65?

Reply:
So, in ATC5401, frames 60 and 65 are "Application program driven. Not scheduled.".
Frame 61 is the long status version of default frame 67, which we have never used.
Both frames 61 and 65 are deprecated in ATC5301.

Reply:
Good point about 60 and 65. Don't think there would be any need to schedule them. I think we should allow 61, for use with ITS cabinets.

Reply:
We will need to add support for frame 61 as well as allowing it to be scheduled. Would be better in a separate commit?

Reply:
Oh, then let's not bother with frame 61 for now. I was thinking that the support was already there.